Misc/NEWS.d/next/Library/2019-09-09-04-43-26.bpo-34037.kX3ZqZ.rst

```diff
@@ -0,0 +1 @@
+For :mod:`asyncio`, add attribute ``asyncio.BaseEventLoop.wait_executor_on_close`` to provide an option for waiting for the executor before closing the event loop. Also adds methods ``loop.get_wait_executor_on_close`` and ``loop.set_wait_executor_on_close``.
```

Comment:
it's not an attribute but 2 new methods.

Reply:
@vstinner:
> it's not an attribute but 2 new methods.

When I initially created the news entry, the ``wait_executor_on_close`` attribute/property (within Python, I believe property would probably be the more correct term) was public, as it was in your PR. But then I realized that it should probably be private since the getter and setter methods were created. 

I'll update it accordingly.